src/public_data/index.md

```diff
@@ -0,0 +1,91 @@
+
+## Accessing Public Data Through the Public HTTP Endpoint
+
+Some BigQuery datasets are also published as gzipped JSON files through the public HTTP endpoint:
+[https://public-data.telemetry.mozilla.org][public_data_endpoint].
+
+A list of available public datasets is available at: [https://public-data.telemetry.mozilla.org/all-datasets.json](https://public-data.telemetry.mozilla.org/all-datasets.json)
+This list contains the names of available datasets, additional metadata and links to the
+storage locations of the files containing the data.
+
+For example:
+
+```json
+{
+    "telemetry_derived": {      // dataset name
+        "deviations": {         // table name
+            "v1": {             // table version
+                "friendly_name": "Deviations",
+                "description": "Deviation of different metrics from forecast.",
+                "incremental": true,
+                "incremental_export": false,
+                "review_link": "https://bugzilla.mozilla.org/show_bug.cgi?id=1624528",  
+                "files_metadata": "https://public-data.telemetry.mozilla.org/api/v1/tables/telemetry_derived/deviations/v1/files/metadata.json",
+                "files_uri": "https://public-data.telemetry.mozilla.org/api/v1/tables/telemetry_derived/deviations/v1/files",
+                "last_updated": "https://public-data.telemetry.mozilla.org/api/v1/tables/telemetry_derived/deviations/v1/last_updated"
+            }
+        },
+        "ssl_ratios": {
+            "v1": {
+                "friendly_name": "SSL Ratios",
+                "description": "Percentages of page loads Firefox users have performed that were  conducted over SSL broken down by country.",
+                "incremental": true,
+                "incremental_export": false,
+                "review_link": "https://bugzilla.mozilla.org/show_bug.cgi?id=1414839",
+                "files_metadata": "https://public-data.telemetry.mozilla.org/api/v1/tables/telemetry_derived/ssl_ratios/v1/files/metadata.json",
+                "files_uri": "https://public-data.telemetry.mozilla.org/api/v1/tables/telemetry_derived/ssl_ratios/v1/files",
+                "last_updated": "https://public-data.telemetry.mozilla.org/api/v1/tables/telemetry_derived/ssl_ratios/v1/last_updated"
+            }
+        },
+        // [...]
+    }
+}
+```
+
+* `incremental`: 
+    * `true`: data gets incrementally updated which means that new data gets added periodically 
+    (for most datasets on a daily basis)
+    * `false`: the entire table data gets updated periodically 
+* `incremental_export`:
+    * `true`: data for each `submission_date` gets exported into separate directories (e.g.
+    `files/2020-04-15`, `files/2020-04-16`, ...)
+    * `false`: all data gets exported into one `files/` directory 
+* `review_link`: links to the Bugzilla bug for the data review
+* `files_metadata`: `metadata.json` contains a list of links to all available data files
+* `files_uri`: location of data files
```

Comment:
Note that the Firefox json viewer makes the files_uri values clickable links, but those links return "not found" errors.

Reply:
Yeah, I guess it would be nice to list files or redirect to the `metadata.json` file